### PR TITLE
update socket.io and socket.io-client packages

### PR DESCRIPTION
**Pull request by amirbey**

### CODE REVIEW

File: api/socketIO.js [hunk 1]
```diff
@@ -15,14 +15,15 @@ if (redisConfig) {
...
-  socketIO.adapter(redisAdapter({ pubClient, subClient }));
-
   pubClient.on('error', (err) => {
     logger.error(`redisAdapter pubClient error: ${err}`);
   });
   subClient.on('error', (err) => {
     logger.error(`redisAdapter subClient error: ${err}`);
   });
+
+  socketIO.adapter(redisAdapter({ pubClient, subClient }));
```
Comment (davemcorwin) [inline api/socketIO.js:25]: does moving this have an effect?

**Review by davemcorwin — CHANGES_REQUESTED** [review]
I think there is more work to do here: https://github.com/socketio/socket.io/releases/tag/3.0.0

> `use` has been removed....

File: api/socketIO.js (hunk 1 above)
Comment (amirbey) [reply api/socketIO.js:25]: not sure ... i was just assigning the error handling before attaching the client to the adapter

Comment (amirbey) [reply api/socketIO.js:25]: it appaers socket.use is no longer used ... but io.use still appears to be used ... in this io is socketIO

**Review by davemcorwin — APPROVED** [review]
LGTM

**Pull request merged**